sdk/cosmos/azure-cosmos/.gitignore

```diff
@@ -0,0 +1,231 @@
+RunTest.cmd
```

Comment:
Perhaps could be part of a future clean-up but do we really need a unique gitignore file under the cosmos directory?

Reply:
I asked this up at the top, I think it is a question for the Cosmos team, I will ping them directly.

Reply:
This can be cleaned up later if necessary I wouldn't block on it for getting the initial code in.